### PR TITLE
API-1835: add test to ensure we dont' accidentally drop a condition during the …

### PR DESCRIPTION
**Pull request by deads2k**
…apply refactor

/assign @bertinatto

### CODE REVIEW

File: test/extended/operators/management_plane_operators.go [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+	}
+)
+
+var _ = g.Describe("[sig-arch][Early] Operators", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithoutNamespace("management-plane-operators")
+
+	g.Describe("low level operators", func() {
+		g.It("should have at least the conditions we had in 4.17", func() {
```
Comment (pacevedom) [inline test/extended/operators/management_plane_operators.go:380]: Would it be possible to tag this with `[apigroup:operator.openshift.io]` to enable skipping in MicroShift?

Comment (pacevedom) [reply test/extended/operators/management_plane_operators.go:379]: Thanks!

**Review by p0lyn0mial — COMMENTED** [review]
LGTM

**Comment by deads2k** [comment]
doh.  victim of validation

> E0926 16:24:24.156948       1 base_controller.go:269] "ResourceSyncController" controller failed to sync "", err: unable to ApplyStatus for operator using fieldManager "ResourceSyncController-reportDegraded": KubeControllerManager.operator.openshift.io "cluster" is invalid: [conditions[19].lastTransitionTime: Required value, <nil>: Invalid value: "null": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation]

**Comment by deads2k** [comment]
/hold cancel

tests out ok.

**Comment by deads2k** [comment]
/override ci/prow/e2e-aws-ovn-serial

timeout, test worked previously

**Comment by deads2k** [comment]
/override ci/prow/e2e-aws-ovn-fips

didn't fail on this

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**